modules/amf-registration/amf-registration-service/service.xml

```diff
@@ -29,7 +29,7 @@
 		<column name="accountName" type="String" />
 		<column name="emailAddress" type="String" />
 		<column name="gender" type="String" />
-		<column name="birthday" type="Date" />
+		<column name="birthday" type="String" />
```

Comment:
you can use a `Date` variable instead of `String`. you can convert it using a `parse` function.